lib/instances/TextButton.lua

```diff
@@ -0,0 +1,24 @@
+local Color3 = import("../types/Color3")
+local GuiButton = import("./GuiButton")
+local InstanceProperty = import("../InstanceProperty")
+local TextButton = GuiButton:extend("TextButton")
+
+TextButton.properties.Text = InstanceProperty.normal({
+	getDefault = function()
+		return ""
```

Comment:
@Kampfkarren does Text/TextColor3/TextSize also need to be updated here?

Reply:
The default value of `Text` for a `TextButton` created via `Instance.new` is `Button`, for reference